src/fragments/lib/logging/common/access_logs/access_logs.mdx

```diff
@@ -0,0 +1,4 @@
+Log messages are sent to the Amazon CloudWatch log group that you created and specified during plugin setup.  Each user log messages can be found in user-specific log stream in the log group.  Log stream names are generated based on the user's device identifier and user identifier
```

Comment:
small tweaks. Also - can we make it clear how the device identifier is created?

Log messages are sent to the Amazon CloudWatch log group that you created and specified [initial set up page].  Each user log messages can be found in user-specific log stream in the log group. Log stream names are generated based on the user's device identifier and a user identifier.

Reply:
Updated the paragraph per suggestion and added additional detail on the device identifier.